ang/prospect/prospect-converted/decorators/case-details.decorators.js

```diff
@@ -0,0 +1,69 @@
+(function (angular, $, _) {
+  var module = angular.module('civicase');
+
+  module.config(function ($provide) {
+    $provide.decorator('civicaseCaseDetailsDirective', civicaseCaseDetailsDirectiveDecorator);
+  });
+
+  function civicaseCaseDetailsDirectiveDecorator ($delegate, ProspectConverted) {
+    var civicaseCaseDetails = $delegate[0];
+
+    civicaseCaseDetails.compile = function () {
+      return function ($scope) {
+        $scope.$on('updateCaseData', updateCaseListener);
+
```

Comment:
Is this space intentional?

Reply:
Yes, because before defining a new function, we leave a blank line.